db/seeds.rb

```diff
@@ -1,7 +1,53 @@
 # This file should contain all the record creation needed to seed the database with its default values.
 # The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
 #
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+# run $ rake db:seed to populate seed data
+
+id = 0
+date = Time.new.strftime("%Y-%m-%d %H:%M:%S")
+
+category_list = ["Nature", "Portraits", "Abstract", "Category4", "Category5"]
+
+# Create 5 categories with 1 painting each
+category_list.each do |name|
+  Category.create( 
+    id: id + 1, 
+    name: name, 
+    created_at: date, 
+    updated_at: date 
+  )
+
+  Painting.create(
+    id: id + 1,
+    title: "Painting#{id}",
+    size: "5x7",
+    medium: "Oil",
+    price: "1000",
+    new: false,
+    sold: false,
+    created_at: date, 
```

Comment:
Rails will [fill in this value automatically](https://guides.rubyonrails.org/active_record_basics.html#schema-conventions). The same applies for `updated_at` which should just be left as `nil` since we're not actually updating anything here